src/it/java/servlets/admin/config/EnableCheatsIT.java

```diff
@@ -1,271 +1,271 @@
-						log.fatal(message);
-						fail(message);
-					}
-				} else {
-					String message = "Admin was unable to enable cheat sheets for all";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-
-	/**
-	 * This test checks that admin users can enable restricted cheat sheets (Admins Only)
-	 */
-	@Test
-	public void testAdminRestrictedEnableCheatsCall()
-	{
-		String userName = "configAdminTester";
-		String password = userName;
-		//Verify / Create user in DB
-		try
-		{
-			TestProperties.verifyTestAdmin(log, applicationRoot, userName, password);
-			//Sign in as Admin User
-			log.debug("Signing in as User Through LoginServlet");
-			TestProperties.loginDoPost(log, request, response, userName, userName, null, lang);
-			log.debug("Login Servlet Complete, Getting CSRF Token");
-			if(response.getCookie("token") == null)
-				fail("No CSRF Token Was Returned from Login Servlet");
-			String csrfToken = response.getCookie("token").getValue();
-			if(csrfToken.isEmpty())
-			{
-				String message = new String("No CSRF token returned from Login Servlet");
-				log.fatal(message);
-				fail(message);
-			}
-			else
-			{
-				String enableForAll = "false";
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost(csrfToken, enableForAll);
-				if(responseBody.contains("Cheat Sheets have been enabled for Security Shepherd Administrators"))
-				{
-					log.debug("Cheat Sheets have been enabled for Security Shepherd Administrators");
-					if(!CheatSheetStatus.isEnabledForAdminsOnly())
-					{
-						String message = "Cheat Sheets were not actually Enabled for admins only";
-						log.fatal(message);
-						fail(message);
-					}
-					else if(CheatSheetStatus.isEnabledForPlayers())
-					{
-						String message = "Cheat Sheets were still enabled for players";
-						log.fatal(message);
-						fail(message);
-					}
-				}
-				else
-				{
-					String message = "Admin was unable to enable cheat sheets for administrators";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-}
+package servlets.admin.config;
+
+import static org.junit.Assert.*;
+
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletConfig;
+
+import testUtils.TestProperties;
+import utils.CheatSheetStatus;
+import utils.InstallationException;
+
+public class EnableCheatsIT
+{
+	private static org.apache.log4j.Logger log = Logger.getLogger(EnableCheats.class);
```

Comment:
EnableCheats.class should also be renamed to EnableCheatsIT (mistake from the Test class)